src/optimise.jl

```diff
@@ -1,17 +1,29 @@
-
+﻿
+export HeapedArray,SortedArray
 """
+    minimise(f, X, str=SortedArray, tol=1e-3) 
+    or
+    minimise(f, X, str=HeapedArray, tol=1e-3)
+    or 
     minimise(f, X, tol=1e-3)
 
-Find the global minimum of the function `f` over the `Interval` or `IntervalBox` `X` using the Moore-Skelboe algorithm.
+Find the global minimum of the function `f` over the `Interval` or `IntervalBox` `X` using the Moore-Skelboe algorithm. By specifing the way in which vector element are kept arranged which is in heaped array or in sorted array. If you not specify any particular strategy to keep vector elements arranged then by default heaped array is used.  
 
 For higher-dimensional functions ``f:\\mathbb{R}^n \\to \\mathbb{R}``, `f` must take a single vector argument.
 
 Returns an interval containing the global minimum, and a list of boxes that contain the minimisers.
 """
-function minimise(f, X::T, tol=1e-3) where {T}
+
+
+defstra=HeapedArray
```

Comment:
This line is unnecessary.

Reply:
I wrote this line just to show that default strategy is `HeapedArray`